src/api.rs

```diff
@@ -86,6 +86,12 @@ impl<'a, T: CronCatInterface> CronCat<'a, T> {
             },
         )
     }
+
+    pub fn purge(&self, task_tags: Vec<String>) -> AbstractSdkResult<CosmosMsg> {
```

Comment:
I was thinking that `purge` would remove all the tasks  that have already been removed by cron-cat itself. Does that make sense?

Reply:
So it would remove all the `removed_tasks` returned by the active task query.

Reply:
> I was thinking that `purge` would remove all the tasks that have already been removed by cron-cat itself. Does that make sense?

Yeah it makes sense, not sure about it considering gas usage. It's still not worth cleaning up state in most cases, so IMO let's keep it this way?

Reply:
Yeah that's the idea, I think it could be something that's called when the user knows they created a task that will only run once and that's unique. 

If you don't think it's worth doing then you don't have to do it, your call :)

Reply:
Free queries 🚀🚀🚀🚀

Reply:
Okey cool :)